### PR TITLE
Cleanup fix

### PR DESCRIPTION
**Pull request by breznak**
- fixes Distractor sampling error made in merging our 2 different codebases #42 
- improves GUI
  - checkboxes removed
  - cleanup & DistractorSettings removed
- regression: I broke the "default button"

### CODE REVIEW

File: src/cz/cvut/cognitive/override/DistractionScreenGUIController.java [hunk 1]
```diff
@@ -175,323 +121,113 @@ public void onStartScreen()
...
+                new WeatherDistraction(sim, 1f, Slider_rain.getValue(), WeatherDistraction.Type.RAIN); 
+                new WeatherDistraction(sim, 1f, Slider_snow.getValue(), WeatherDistraction.Type.SNOW); 
+                new WeatherDistraction(sim, 1f, Slider_fog.getValue(), WeatherDistraction.Type.FOG); 
+                
+                super.clickStartButton();
 	}
         
-          public void clickDefButton()
+        @NiftyEventSubscriber(id="DefaultButton")
+          private void clickDefButton() //FIXME I broke the default button - GUI is not reset
```
Comment (breznak) [inline src/cz/cvut/cognitive/override/DistractionScreenGUIController.java:211]: @JohnnyMarek I broke the DefaultButton, even though it should work. Also the values are shown initially only for some. Can you take a look please?

**Comment by JohnnyMarek** [comment]
will fix it today

**Comment by breznak** [comment]
@JohnnyMarek please make the wheel running by tmr, that's a priority! :)

**Comment by JohnnyMarek** [comment]
👍 will do

**Pull request merged**